### ISSUE TITLE
[r56] Docked Event Sheets Turn Uneditable Dark Gray

### ISSUE THREAD

**Issue by wiverson** [Scirra/Construct-3-bugs#827]
## Problem description

Copy-pasting an event in an event sheet docked below a layout causes the event sheet to turn dark gray.

<img width="1106" alt="screen shot 2017-09-20 at 4 09 46 pm" src="https://user-images.githubusercontent.com/873850/30671958-82787e0e-9e1e-11e7-81c2-95def1bda5de.png">

## Attach a .c3p

Easier to repro with new empty project.

## Steps to reproduce

1. New project.
2. Open layout.
3. Open event sheet, dock event sheet to window below the layout.
4. Create an event (e.g. System -> Every tick).
5. Copy event.
6. Paste event.

## Observed result

Event sheet turns dark gray.

Forcing a refresh to the event sheet seems to fix, e.g. resizing the event sheet or closing & reopening.

## Expected result

Just to see the event sheet update properly.

## Affected browsers

* Chrome: yes
* Firefox: ?
* Edge: ?
* Safari: ?

## System details

<details><summary>View Details...</summary>
Platform information
Browser: Chrome
Browser version: 60.0.3112.113
Browser engine: Blink
Browser architecture: (unknown)
Context: browser
Operating system: Mac OS X
Operating system version: 10.12_6
Operating system architecture: (unknown)
Device type: desktop
Device pixel ratio: 2
Logical CPU cores: 8
Approx. device memory: (unavailable)
User agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36
C3 release: r56
Language setting: en-US

Local storage
Storage quota (approx): 30 gb
Storage usage (approx): 1.3 gb (4.4%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Vendor: ATI Technologies Inc.
Renderer: AMD Radeon R9 M290X OpenGL Engine
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 2048
Extensions:

EXT_color_buffer_float
EXT_disjoint_timer_query_webgl2
EXT_texture_filter_anisotropic
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

**Comment by AshleyScirra** [comment]
Does this reproduce with the default theme or is it specific to the dark theme?

**Comment by wiverson** [comment]
Happens in the default theme as well, although it turns white instead of dark grey.

**Comment by shortercode** [comment]
Noticed this bug look a bit forgotten, I've managed to reproduce this but only on the mac. Still seems to happen in the current version as well.

**Comment by shortercode** [comment]
Well it's a pretty awkward bug, I've tried a few hacks to try and workaround it and none have worked 100%. I've added something to r77 which will prevent it from happening ( most of the time ). It's about as good as it's going to get, until chrome fix it. So going to close the issue now.

If anyone is suffering from this issue in future, I recommend either using a different browser or just placing the pane to the left/right of the layoutview instead.
